Yona/Yona/StringExtensions.swift

```diff
@@ -41,16 +41,24 @@ extension String {
         
     }
     
-    func validateMobileNumber() -> Bool {
-        let character  = CharacterSet(charactersIn: "+0123456789").inverted
-        let inputString:NSArray = self.components(separatedBy: character) as NSArray
-        let filtered = inputString.componentsJoined(by: "")
-        
-        if (self != filtered || self.characters.count == 0 || self.characters.count <= YonaConstants.mobilePhoneLength.netherlands || self == "+31") {
-            return false
-        } else {
-            return true
+    func isValidMobileNumber() -> Bool {
+        let PHONE_REGEX = "^(\\+)[0-9]{6,20}$"
```

Comment:
Why are you having parenthesis in the regex?

Reply:
Removed parenthesis